Test/AwsCommonRuntimeKitTests/io/RetryerTests.swift

```diff
@@ -0,0 +1,50 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import XCTest
+@testable import AwsCommonRuntimeKit
+
+class RetryerTests: CrtXCBaseTestCase {
+    let expectation = XCTestExpectation(description: "Credentials callback was called")
+    
+    func testCreateAWSRetryer() throws {
+        let shutDownOptions = ShutDownCallbackOptions { semaphore in
+            semaphore.signal()
+        }
+        let elg = EventLoopGroup(allocator: allocator, shutDownOptions: shutDownOptions )
```

Comment:
if tests run in parallel, maybe specify 1 thread instead of relying on the default here, otherwise, you're gonna thrash the machine when it runs its tests.

Reply:
changed, good point